components/renku_data_services/notebooks/api/schemas/cloud_storage.py

```diff
@@ -231,22 +231,32 @@ def config_string(self, name: str) -> str:
         if not self.configuration:
             raise ValidationError("Missing configuration for cloud storage")
 
-        # Transform configuration for polybox, switchDrive or sftp
+        # TODO Use RCloneValidator.get_real_configuration(...) instead.
+        # Transform configuration for polybox, switchDrive, openbis or sftp
         storage_type = self.configuration.get("type", "")
         access = self.configuration.get("provider", "")
 
-        if storage_type == "sftp":
-            # Do not allow retries for sftp
-            # Reference: https://rclone.org/docs/#globalconfig
-            self.configuration["override.low_level_retries"] = 1
-
         if storage_type == "polybox" or storage_type == "switchDrive":
             self.configuration["type"] = "webdav"
             self.configuration["provider"] = ""
             # NOTE: Without the vendor field mounting storage and editing files results in the modification
             # time for touched files to be temporarily set to `1999-09-04` which causes the text
             # editor to complain that the file has changed and whether it should overwrite new changes.
             self.configuration["vendor"] = "owncloud"
+        elif storage_type == "s3" and access == "Switch":
+            # Switch is a fake provider we add for users, we need to replace it since rclone itself
+            # doesn't know it
+            self.configuration["provider"] = "Other"
+        elif storage_type == "openbis":
+            self.configuration["type"] = "sftp"
+            self.configuration["port"] = "2222"
+            self.configuration["user"] = "?"
```

Comment:
question: Why not remove the user. Or is "?" really the right value for the user?